dev-python/dbuspython/dbuspython-1.2.14.recipe

```diff
@@ -0,0 +1,78 @@
+SUMMARY="The original dbus implementation for Python"
+DESCRIPTION="dbus-python is the original Python binding \
+for dbus, the reference implementation of the D-Bus protocol."
+HOMEPAGE="https://www.freedesktop.org/wiki/Software/DBusBindings/#python"
+COPYRIGHT="The D-Bus maintainers"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://dbus.freedesktop.org/releases/dbus-python/dbus-python-1.2.14.tar.gz"
+CHECKSUM_SHA256="b10206ba3dd641e4e46411ab91471c88e0eec1749860e4285193ee68df84ac31"
+SOURCE_DIR="."
```

Comment:
Do you need this "."?

Reply:
SOURCE_DIR="" should also work

Reply:
Should be:
```
SOURCE_DIR:"dbus-python-$portVersion"
```

Reply:
I completely removed the entire parameter, the build still passed and now looks more similar to the other `.recipe` files within the `dev-python` directory.

The `setup.py` file resides on the root of the directory of the untarred folder.

My build kept failing, so I added that option while I was experimenting. It was a matter of a leftover parameter.